techhive/server.js

```diff
@@ -245,6 +277,35 @@ app.get('/users/:id/saved-internships', async (req, res) => {
   }
 });
 
+async function sendNotificationToStudents(internshipLink) {
```

Comment:
The code in this function looks the same as what's above in the try-block of the posting internships route. 
Why the duplication? Can we call `sendNotificationToStudents()` directly above?